nucliadb/nucliadb/ingest/orm/abc.py

```diff
@@ -67,3 +69,15 @@ async def del_vectorset(self, shard: str, id: str):
     @abstractmethod
     async def set_vectorset(self, shard: str, id: str):
         pass
+
+    async def stream_get_fields(
+        self, stream_request: StreamRequest
+    ) -> AsyncIterator[DocumentItem]:
+        async for idandfacets in self.reader.Documents(stream_request=stream_request):
+            yield idandfacets
+
+    async def stream_get_paragraphs(
+        self, stream_request: StreamRequest
+    ) -> AsyncIterator[ParagraphItem]:
+        async for idandfacets in self.reader.Paragraphs(stream_request=stream_request):
+            yield idandfacets
```

Comment:
The naming here is confusing:

- function is called `stream_get_fields`
- `reader.Documents` is called
- we return `idandfacets`

I am not sure what it should be, but can you improve it?

Reply:
DocumentItem should be changed , out of the scope of this PR